zipline/finance/slippage.py

```diff
@@ -314,10 +314,13 @@ def __init__(self, spread=0.0):
         self.spread = spread
 
     def process_order(self, event, order, dt):
+
+        price = self.data_portal.get_current_price_data(order.sid, 'close')
```

Comment:
It is 'hotwired' in in tradesimulation, which was pre-existing to this patch. (VolumeShare also uses the data_portal.)

We should pass it explicitly.